README.md

```diff
@@ -235,9 +235,9 @@ will precede the `go` command to start thinking with `sd 5`. The other `go_comma
   - `allow_matchmaking`: Whether to challenge other bots.
   - `challenge_variant`: The variant for the challenges. If set to `random` a variant from the ones enabled in `challenge.variants` will be chosen at random.
   - `challenge_timeout`: The time (in minutes) the bot has to be idle before it creates a challenge.
-  - `challenge_initial_time`: The initial time (in seconds) for the challenges.
-  - `challenge_increment`: The increment (in seconds) for the challenges.
-  - `challenge_days`: The days for a correspondence challenge. If this option is enabled, a correspondence challenge will be created even if `challenge_initial_time` is enabled.
+  - `challenge_initial_time`: The initial time (in seconds and to be chosen at random) for the challenges.
+  - `challenge_increment`: The increment (in seconds and to be chosen at random) for the challenges.
+  - `challenge_days`: The days for a correspondence challenge (to be chosen at random). If this option is enabled, a correspondence challenge will be created even if `challenge_initial_time` is enabled.
```

Comment:
The descriptions here should note that a list of times is expected. Otherwise, "chosen at random" doesn't make sense.